### PR TITLE
Add to 4.6 Release Notes: Enhancements to Red Hat Virtualization (RHV…

### PR DESCRIPTION
**Pull request by rolfedh**
…) full stack installer

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -450,6 +450,20 @@ Note the following restrictions for {product-title} on IBM Power Systems:
 * iSCSI
 * 4k Disk support
 
+[id="ocp-4-6-enhancements-to-rhv-installer"]
+==== Enhancements to {rh-virtualization-first} full-stack installer
+
+* Dynamically provision storage {rh-virtualization} storage domains to an {product-title} cluster with a {rh-virtualization} CSI operator.
```
Comment (peterclauterbach) [inline release_notes/ocp-4-6-release-notes.adoc:456]: provision storage _from_ {rh-virtualization} storage domains

**Review by peterclauterbach — CHANGES_REQUESTED** [review]
LGTM, minor edit.

**Review by peterclauterbach — APPROVED** [review]
LGTM

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -450,6 +450,20 @@ Note the following restrictions for {product-title} on IBM Power Systems:
...
 * 4k Disk support
 
+[id="ocp-4-6-enhancements-to-rhv-installer"]
+==== Enhancements to {rh-virtualization-first} full-stack installer
+
+* Dynamically provision storage from {rh-virtualization} storage domains to an {product-title} cluster with a {rh-virtualization} CSI operator.
+* Improved control of workloads and resources by auto-scaling additional {rh-virtualization} virtual machine worker nodes.
+* Support for disconnected or restricted installations with a local mirror: Beneficial for financial, public sector, and secure environments.
+* Install {product-title} on {rh-virtualization} with user-provisioned infrastructure such as an external load balancer. This process uses a series of Ansible playbooks to enable a more flexible installation.
+* {product-title} version 4.6 requires {rh-virtualization} version 4.4.2+.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-6-release-notes.adoc]: s/4.4.2+/4.4.2 or higher

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -450,6 +450,20 @@ Note the following restrictions for {product-title} on IBM Power Systems:
...
++
+[IMPORTANT]
+====
+If you are running {product-title} version 4.5 on {rh-virtualization} version 4.3, upgrade {rh-virtualization} to version 4.4.2+ before updating {product-title} to version 4.6.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-6-release-notes.adoc]: s/4.4.2+/4.4.2 or higher

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -450,6 +450,20 @@ Note the following restrictions for {product-title} on IBM Power Systems:
 * iSCSI
 * 4k Disk support
 
+[id="ocp-4-6-enhancements-to-rhv-installer"]
+==== Enhancements to {rh-virtualization-first} full-stack installer
+
+* Dynamically provision storage from {rh-virtualization} storage domains to an {product-title} cluster with a {rh-virtualization} CSI operator.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-6-release-notes.adoc:456]: "You can now dynamically..." perhaps?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -450,6 +450,20 @@ Note the following restrictions for {product-title} on IBM Power Systems:
...
+* Improved control of workloads and resources by auto-scaling additional {rh-virtualization} virtual machine worker nodes.
+* Support for disconnected or restricted installations with a local mirror: Beneficial for financial, public sector, and secure environments.
+* Install {product-title} on {rh-virtualization} with user-provisioned infrastructure such as an external load balancer. This process uses a series of Ansible playbooks to enable a more flexible installation.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-6-release-notes.adoc:459]: "You can now install..." perhaps?

**Review by jeana-redhat — COMMENTED** [review]
A few minor suggestions

**Pull request merged**